### DISCUSSION TITLE
Allow adding metadata for product variants

### DISCUSSION THREAD

**Discussion by mrguruo** [medusajs/medusa#9749] [Feature Requests]
The ability to add metadata for product variants would be very helpful! Different variants will often have properties that need to be displayed to customers, but at the moment there is no way to add this information in the admin UI.

**Comment by julianengel** [comment]
Seconding this. I think it's possible via API, but would be nice to do in the admin UI.

**Comment by kasperkristensen** [comment]
Hi @mrguruo, thanks for the suggestion, its an oversight on our part. I have just opened a PR to add this: https://github.com/medusajs/medusa/pull/10285. Should be part of the next release.
